artifacts_test.py

```diff
@@ -227,6 +230,23 @@ def verify_docker_locale_settings(self) -> None:
             configured_value = locale_settings.get(setting_key)
             self.assertEqual(configured_value, expected_value)
 
+    def verify_docker_latest_match_release(self) -> None:
+        for product in ['scylla', 'scylla-enterprise']:
```

Comment:
You can use `typing.get_args(ScyllaProduct)` here.

Reply:
nice, didn't know about that...

Reply:
fixed